pkg/utils/refutils/object_ref.go

```diff
@@ -0,0 +1,21 @@
+package refutils
```

Comment:
Do we really need a whole knew package for this, why not just put it in `ezkube`?